htdocs/bom/class/bom.class.php

```diff
@@ -1806,6 +1785,7 @@ public function __construct(DoliDB $db)
 		global $conf, $langs;
 
 		$this->db = $db;
+		$this->setPicto('bomline');
```

Comment:
because if we juste use $this->picto = xxx php consider this as an overdie of the fonction if done into class description or class constructor.
to avoid override we need tu use a parent fonction of the class.
this is what i've found about the best practice in PHP 8.2.